pmlc/dialect/pxa/transforms/passes.h

```diff
@@ -13,10 +15,24 @@ class OpPassBase;
 
 namespace pmlc::dialect::pxa {
 
+struct StencilPassOptions
+    : public mlir::PassPipelineOptions<StencilPassOptions> {
+  Option<int> numberOfThreadsOption{
+      *this, "affine-stencil-number-threads",
+      llvm::cl::desc("Specifies number of threads for the stencilling pass")};
+};
+
 struct StencilPass : public mlir::FunctionPass<StencilPass> {
+  StencilPass() = default;
+  StencilPass(const StencilPass &) {}
+  explicit StencilPass(const StencilPassOptions &options);
+
+  Option<int> numberOfThreadsOption{
+      *this, "affine-stencil-number-threads",
```

Comment:
No need to prefix `affine-stencil`, just use `threads`.

Reply:
Fixed